src/pug/blog/typing_methods.pug

```diff
@@ -0,0 +1,425 @@
+  table.manual
+    tr
+      th Consideration 11
+      td Given similar complexity, methods that allow for faster typing will be considered better.
+
+  hr
+  h3 3. Generic solutions
+
+  h4 3.1 Mapping of controls
+  p Since a controller has significantly fewer buttons than a keyboard, we have to find alternative ways to map characters to these limited buttons or analog control surfaces.
+  ul
+    li #[em Grouping] several characters, for example #[em A], #[em B], #[em C] into the a single physical button or selectable, then choosing one of these characters by:
+    ul
+      li Pressing the button repeatedly during a fixed amount of time to cycle through the characters, as was one common solution on feature phones (aka writing SMS messages in the 2000s) #[i (fig. A)]. Although this conflicts with #[em consideration 7].
+      li Using simultaneously a button or control surface to preselect a character group, then using another button or control surface to cycle or select the final character. As for example in the Steam Controller #[em Daisywheel] #[i (fig. E)].
+    li Using the analog surfaces to allow input in which each character is clearly distinguishable, as for example #[em character recognition].
+
+  +figure(
+    'A',
+    'Nokia 3210 keypad (1999)',
+    '/static/blog/keypad.jpg',
+    'small inline'
+  )
+  +figure(
+    'E',
+    'Steam Controller Daisywheel (2015)',
+    '/static/blog/daisywheel_restoration.png',
+    'small inline'
+  )
+
+  h4 3.2. Characters preview and commit
+  p Since we do not want to rely on OS integration (#[em (consideration 3]), one of the potential solutions we tried is to apply the concept of #[em character preview and commit] that was used in #[em feature phones], in which a single button or action is used for cycling between several characters, and only later the character is "committed" when certain conditions are met (other button is pressed, or after a defined period of time).
+  +figure(
+    'F',
+    'Character preview and commit',
+    '/static/blog/sms.gif',
+    'small'
+  )
+  p This same concept can be used in PC by sending (and deleting) characters on demand. Unfortunately in our tests we found that this solution  it is #[em not compatible with 100%] of the PC use cases, as for example it may be problematic:
+  ul
+    li If a searchbox is auto-completing what is written on it, and "auto-committing" characters.
+    li If searchbox suggestions rendering is making the system to get unresponsive.
+    li If the input is obscured as in a password field.
+    li If the UI is expecting a simple key press (binding menu).
+
+  hr
+  h3 4. Specific solutions
+  p We evaluated a total of #[em 9 potential typing methods], and we created early implementations for 7.
```

Comment:
Why emphasize the 9?
Only 7 are described here. Where is the evaluation of the other two (and why they were dropped)?
Only 5 were implemented (qwerty rows and gyro char recognition were dropped)